### DISCUSSION TITLE
vitepress 侧边栏无法翻译

### DISCUSSION THREAD

**Discussion by zsd8111** [community/community#88508] [Other Feature Feedback, Questions, & Ideas]
### The sidebar cannot be translated

Question

### Body

 locales: {

    root: {
      label: '简体中文',
      lang: 'zh'
    },
    en: {
      label: 'English',
      lang: 'en',
    }
  },
<img width="883" alt="8131c5334e25d473133b29cb0f37c20" src="https://github.com/community/community/assets/76585898/d8899465-bc8e-4491-bac8-ae7b7aea0e30">
<img width="275" alt="daa323922771ba38a54eef70258803a" src="https://github.com/community/community/assets/76585898/bf88ce08-9850-452c-b79f-32d8b53ff799">

**Comment by queenofcorgis** [comment]
Hi @zsd8111 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
